src/components/table/mixins/sort.js

```diff
@@ -21,10 +21,26 @@ export default {
       }
 
       return [...this.data].sort((a, b) => {
-        const aValue = a[this.sortBy]
-        const bValue = b[this.sortBy]
+        let aValue = a[this.sortBy]
+        let bValue = b[this.sortBy]
         const modifier = this.sortDir === SORT_DIRECTION_ASC ? 1 : -1
 
+        const { sortable } = this.fields.find(f => f.name === this.sortBy)
+
+        if (sortable) {
+          if (typeof sortable === "string") {
+            const genArrComp = vals => {
+              return vals.map(val => val[sortable]).join("")
+            }
+
+            aValue = genArrComp(aValue)
+            bValue = genArrComp(bValue)
+          } else if (typeof sortable === "function") {
```

Comment:
can we add one test for function sortable?

Reply:
Agreed that we should, do you have any idea of a use case?

Reply:
Maybe a star ⭐️ rating ?

Reply:
The branch is merged but I raised an issue [here](https://github.com/owncloud/owncloud-design-system/issues/1437) if you want to get funky :)